libs/community/langchain_community/vectorstores/tencentvectordb.py

```diff
@@ -63,6 +79,63 @@ def __init__(
         self.params = params
 
 
+class MetaField(BaseModel):
+    """MetaData Field for Tencent vector DB."""
+
+    name: str
+    description: Optional[str]
+    data_type: Union[str, Enum]
+    index: bool = False
+
+    def __init__(self, **data: Any) -> None:
+        super().__init__(**data)
+        enum = guard_import("tcvectordb.model.enum")
+        if isinstance(self.data_type, str):
+            if self.data_type not in META_FIELD_TYPES:
+                raise ValueError(f"unsupported data_type {self.data_type}")
+            target = [
+                fe
+                for fe in enum.FieldType
+                if fe.value.lower() == self.data_type.lower()
+            ]
+            if target:
+                self.data_type = target[0]
+            else:
+                raise ValueError(f"unsupported data_type {self.data_type}")
+        else:
+            if self.data_type not in enum.FieldType:
+                raise ValueError(f"unsupported data_type {self.data_type}")
+
+
+def is_valid_embedding(embedding: Embeddings) -> bool:
+    return embedding is not None and not isinstance(
+        embedding, (FakeEmbeddings, DeterministicFakeEmbedding)
```

Comment:
why do we need to check this?

Reply:
The purpose of FakeEmbedding is solely for demo purposes, and Tencent Cloud VectorDB allows the option not to specify LangChain Embeddings models (the alternative is to specify an embedding model name built into Tencent Cloud VectorDB). Therefore, we do not recommend explicitly using FakeEmbedding for demos, and thus classify FakeEmbedding as an illegal embedding.

If you believe it's necessary to keep the FakeEmbedding for demonstration purposes, then I can modify the code accordingly. What do you think?

Reply:
using fake embeddings can also be useful for testing purposes

Reply:
OK, fixed it.

Reply:
fixed all the CI issue, let's try again @baskaryan

Reply:
Is my fault，find_spec can throw a ModuleNotFoundError。 fixed it now. @baskaryan